website/docs/enterprise/getting-started/releases-enterprise.mdx

```diff
@@ -10,6 +10,44 @@ import TierLabel from "../../_components/TierLabel";
 This page details the changes for Weave GitOps Enterprise and its associated components. For Weave GitOps OSS, please see the release notes on [GitHub](https://github.com/weaveworks/weave-gitops/releases).
 :::
 
+## v0.37.0
+2023-11-23
+
+### Highlights
+
+##### Explorer enabled by default. GitOpsSets and GitOpsTemplates uses Explorer by default.
+
+Explorer is enabled by default for GitOpsSets and GitOpsTemplates. This improves the pages with:
+ - Showing GitOpsSets on multiple clusters
+ - Faster page load and full-text search for enhanced . 
+
+You could control where Explorer is enabled via [configuration](https://docs.gitops.weave.works/docs/explorer/getting-started/#setup). 
+
+##### Explorer RED metrics
+
+RED metrics and corresponding Grafana dashboard panels have been added to the objects cleaner component of Explorer, enhancing the troubleshooting capabilities in Explorer's management path.
+
+### Enhancements
+- Added new tab for events in the Cluster Discovery view
+- Added a new component for metadata in the Automated Cluster Discovery view
+- Enhance filter headings style for better readability
+
+### Bug Fixes
+- Support dark mode on Explorer pagination controls
+- Fixes AutomatedClusterDiscovery deletion
+- Fix cluster creation URL bug with explorer
+- Bootstrap CLI: Added validation case for repo url without scheme
+- Add Yaml tab to Pipeline details
```

Comment:
I could see the enhancements and bug fixes section. we dont usually have them  as stated in [here](https://github.com/weaveworks/weave-gitops-enterprise/blob/main/docs/releasing.md?plain=1#L94) ... see also past releases that only have `highlights`, `breaking changes`, etc ... 

(Btw, I see there is a case for bringing bug fixes to the release notes as our ee users would benefit from but raising the point as the release notes for this version wont be consistent with previous versions)

Reply:
I've removed any technical enhancements or bug fixes from the original release notes, just including relevant user updates. I belive both enhancements and bug fixes should be included moving forward.

Reply:
👍  +1 on it, i guess that then we could reflect that expectation in the [releasing doc](https://github.com/weaveworks/weave-gitops-enterprise/blob/main/docs/releasing.md) where suits. 

(Btw, feel free to extend this one  https://github.com/weaveworks/weave-gitops-enterprise/pull/3677/files  which is pending of review)

Reply:
added an edit to the process PR. https://github.com/weaveworks/weave-gitops-enterprise/pull/3677/files#r1409134334